mapviz/src/mapviz.cpp

```diff
@@ -1099,8 +1066,10 @@ void Mapviz::Hover(double x, double y, double scale)
 
       lat_lon_text += ", ";
 
-      double lon_scale = (1.0 / (111111.0 * std::cos(point.y() * swri_math_util::_deg_2_rad))) * scale;
-      int32_t lon_precision = static_cast<int32_t>(std::ceil(std::max(0.0, std::log10(1.0 / lon_scale))));
+      double lon_scale = (1.0
```

Comment:
My gut feeling is that the goal here is to determine exactly how precise our floating point numbers are so that the display can be truncated after the decimal place where we lose precision, but I don't quite follow the math, and the person who original wrote this is no longer around to answer...